src/_data/data.js

```diff
@@ -489,6 +492,33 @@ function crossReferenceListsAndFaqs(lists, faqs) {
   });
 }
 
+function createUnlistedCategory(faqs, lists) {
+  const unlistedFaqs = faqs.filter( unlistedFaq => unlistedFaq.relatedLists.length === 0 );
+
+  // For system-generated categories, use placeholder values
+  // Use epoch date (Jan 1, 1970) to indicate system-generated content
+  const systemDate = new Date(0);
+
+  const unlistedCategory = {
+    type: 'list',
+    id: 'unlisted',
+    title: 'Unlisted FAQs',
+    icon: '📋',
+    description: 'FAQs not yet assigned to any category',
+    faqs: unlistedFaqs,
+    permalink: '/faq/unlisted/',
+    isSystemGenerated: true,
+    editOnGithubUrl: '',
```

Comment:
I think it would make more sense to not have `editOnGithubUrl` at all and add a guard in the templates.